src/components/MembersPanel.tsx

```diff
@@ -368,9 +368,31 @@ const MembersPanel = ({ members, setMembers }: MembersPanelProps) => {
       );
    }
 
+   function totalMembers(): number {
+      return (
+         members.length
+      );
+   }
+
+   function activeMembers(): number {
+      let activeTotal = 0
+
+      for (let member of members) {
+         if (member.active) {
+            activeTotal += 1
+         }
+      }
+      return (
+         activeTotal
+      );
+   }
+
    return (
       <Paper sx={{ p: 2, backgroundColor: "Gainsboro" }}>
-         <Typography variant="h6">Members:</Typography>
+         <Box sx={{ display: "flex", justifyContent: "space-between" }}>
+            <Typography variant="h6">Members:</Typography>
+            <Typography variant="h6">Active/Total:{activeMembers()}/{totalMembers()}</Typography>
```

Comment:
I think the "Active/Total:" is self-explanatory, so let's only display: activeMembers / totalMembers.
You can also replace the totalMembers() function call with my suggestion above.